pom.xml

```diff
@@ -25,6 +25,32 @@
         <module>javaparser-testing</module>
     </modules>
 
+  <profiles>
+    <profile>
+      <id>default-tools.jar</id>
+      <activation>
+        <property>
+          <name>java.vendor</name>
+          <value>Oracle Corporation</value>
+        </property>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>com.sun</groupId>
+          <artifactId>tools</artifactId>
+          <version>1.7</version>
```

Comment:
This dependency was inserted just because I have adopted JDK7. If I have to change the AJC files to comply to JDK 6, this will also be reflected here.

Moreover, I have edited the pom.xml file poorly, given my lack of experience with maven. Probably there's a better way to add tools.jar to the javac's classpath.